cloudbridge/cloud/providers/gce/resources.py

```diff
@@ -896,8 +896,8 @@ def image_id(self):
             return None
         for disk in self._gce_instance['disks']:
             if 'boot' in disk and disk['boot']:
-                return self._provider.get_gce_resource_data(
-                    disk['source']).get('sourceImage')
+                disk_resource = self._provider.parse_url(disk['source']).get()
```

Comment:
The method name get() seems a bit ambiguous when reading the code. It's not immediately obvious what it is that it's retrieving. Should it be something like get_resource() instead?

Reply:
Done